Modules/posixmodule.c

```diff
@@ -5228,6 +5310,59 @@ os_spawnve_impl(PyObject *module, int mode, path_t *path, PyObject *argv,
 #endif /* HAVE_SPAWNV */
 
 
+#ifdef HAVE_FORK
+/*[clinic input]
+os.register_at_fork
+
+    func: object
```

Comment:
bikeshed: `callable` is the name of a built-in, I wouldn't use it as a parameter name. `function` perhaps.